src/preload.ts

```diff
@@ -89,6 +89,12 @@ window.removeTable = (tableName: string, signal: AbortSignal) =>
     return result;
   });
 
+window.buildSampleCardEncoded = (tableName: string, aidColumns: string[], signal: AbortSignal) =>
+  newTask(signal, async (taskId) => {
```

Comment:
We don't support cancellation, but I'm ok with having it consistent.

Reply:
yeah, I remembered, but chose consistent. Let's fix this in one fell swoop "later"